edns.go

```diff
@@ -103,15 +103,12 @@ func (rr *OPT) SetVersion(v uint8) {
 
 // ExtendedRcode returns the EDNS extended RCODE field (the upper 8 bits of the TTL).
```

Comment:
Thanks.. Should we update the function doc as well? To make it clearer what we're doing here?

Reply:
> Thanks.. Should we update the function doc as well? To make it clearer what we're doing here?

I agree. We should improve the docs.

And sorry for the delay.
I will submit another PR to improve the docs several days later.